src/main/java/org/opentestsystem/delivery/logging/EventParser.java

```diff
@@ -0,0 +1,54 @@
+package org.opentestsystem.delivery.logging;
+
+import com.google.common.base.Optional;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.opentestsystem.delivery.logging.EventLogger.BaseEventData.HTTP_REQUEST_PARAMETERS;
+import static org.opentestsystem.delivery.logging.EventLogger.BaseEventData.HTTP_SESSION_ID;
+
+/**
+ * Map interceptor request to EventInfo value object used to log event.
+ * Common fields such a http session and request parameters are stored in the returned EventInfo
+ */
+public class EventParser {
+
+  /**
+   * The default pre request parser.
+   *
+   * @param request
+   * @return
+   */
+  public Optional<EventInfo> parsePreHandle(final HttpServletRequest request, final Object handler, final EventLogger logger) {
+    return Optional.of(EventInfo.create(request.getPathInfo(), getEventDataFields(request)));
```

Comment:
The check points are being explicitly set in the interceptor (and exception handlers) after the calls to parsePreHandle and parsePostHandle.  I'll remove setting the EXIT checkpoint in the EventParser.